generators/server/templates/src/test/java/package/web/rest/LogoutResourceIT.java.ejs

```diff
@@ -18,25 +18,15 @@
 -%>
 package <%= packageName %>.web.rest;
 
-import <%= packageName %>.<%= mainClass %>;
+import <%= packageName %>.IntegrationTest;
 import <%= packageName %>.security.AuthoritiesConstants;
-<%_ if (cacheProvider === 'redis') { _%>
-import <%= packageName %>.RedisTestContainerExtension;
-<%_ } _%>
-<%_ if (databaseType === 'neo4j') { _%>
-import <%= packageName %>.AbstractNeo4jIT;
-<%_ } _%>
-<%_ if (reactiveSqlTestContainers) { _%>
-import <%= packageName %>.ReactiveSqlTestContainerExtension;
-<%_ } _%>
 import <%= packageName %>.config.TestSecurityConfiguration;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-<%_ if (databaseType === 'neo4j' || cacheProvider === 'redis' || (reactive && ['mysql', 'postgresql', 'mssql'].includes(prodDatabaseType))) { _%>
+<%_ if (reactive && ['mysql', 'postgresql', 'mssql'].includes(prodDatabaseType)) { _%>
```

Comment:
I believe this can be removed too, this condition is the same as 'reactiveSqlTestContainers'

Reply:
Agreed, its an unused import.